project/EnsimeBuild.scala

```diff
@@ -58,6 +58,7 @@ object EnsimeBuild extends Build with JdkResolver {
 
   lazy val commonSettings = scalariformSettings ++ basicSettings ++ Seq(
     //resolvers += Resolver.sonatypeRepo("snapshots"),
+    resolvers += "Spring IO" at "http://repo.spring.io/libs-release-remote",
```

Comment:
ouch, this is going to need to be added to the `build.sbt` in emacs, vim, etc etc.